### ISSUE TITLE
Assign Container Directly to Bookmarks (Consistent though Editable)

### ISSUE THREAD

**Issue by MichaelTunnell** [mozilla/multi-account-containers#1142]
I think the ability of assigning _Containers_ **directly to Bookmarks** would be absolutely amazing and solve a lot of headaches for a lot of people.

I want to open a single website with multiple sessions, which currently is a pain to do. if I could assign a container to a bookmark then I could accomplish this with so little effort that it would fundamentally change the way I use a web browser to the point that Firefox would take over a wide variety of tasks.

Firefox is my main browser and has been for over a decade but with this it could replace all need for additional browsers.

For example, what about having a _Registered Protocol_ for each container?

Let's say a user has 4 containers, this could be done with IDs like "1, 2, 3, 4" or names like "Personal, Work, TuxDigital, Self-Promotion"

Then you could make a bookmark like this `container-2:http://example.com/page-slug` via ID# or even name like this `container-work:http://example.com/page-slug`

I think the ability to have relatively permanent way to open a bookmark in a particular container would be absolutely game changing.

please add labels: enhancement, vote for me

**Comment by stoically** [comment]
Duplicate where you could vote: https://github.com/mozilla/multi-account-containers/issues/323

**Comment by MichaelTunnell** [comment]
It is different because the main purpose of that thread was the Context Menu feature . . . there is even a label associated to it exclusively for the context menu.

I created this so there would be a rough idea of how many people who are interested in this specifically not just as a secondary mention of a different idea. Yes he wanted both but the developers focused in on the context menu because that was the main request. Making a separate request allows more data as to how many wanted this rather than it included in the context menu data. For example, I couldn't care less about the context menu option if this was available instead. Even though I voted for that, I really wanted the secondary point but as stated, it was the context menu that took precedence on the issue.
